src/test/Fake.Core.UnitTests/Fake.Tools.Pickles.fs

```diff
@@ -0,0 +1,73 @@
+module Fake.Tools.Pickles.Tests
+
+open System.IO
+open Fake.Core
+open Fake.Testing
+open Fake.Tools
+open Expecto
+
+let runCreateProcess setParams =
+  let cp =
+    Pickles.createProcess 
+      (fun param -> 
+        { setParams param with
+            ToolPath = Path.Combine("pickles", "pickles.exe") } )
+  
+  let file, args =
+    match cp.Command with
+    | RawCommand(file, args) -> file, args
+    | _ -> failwithf "expected RawCommand"
+    |> ArgumentHelper.checkIfMono
+    
+  let expectedPath = Path.Combine("pickles", "pickles.exe")
+  Expect.equal file expectedPath "Expected pickles.exe"
+  
+  expectedPath, cp.Command.CommandLine
```

Comment:
this needs to use the 'args' variable and not 'cp' in order to work with mono environments...

I think '(RawCommand(file, args)).CommandLine' should work

Can you please change the other ones as well, this should make the build green again